Packs/CommonScripts/Scripts/CreateIndicatorsFromSTIX/CreateIndicatorsFromSTIX_test.py

```diff
@@ -15,10 +12,35 @@ def test_create_indicators_loop(mocker):
     Then:
         - Validate the indicators extract without errors.
     """
+    args = {}
     with open('test_data/expected_result_example3.json') as json_f:
         indicators = json.load(json_f)
     mocker.patch.object(demisto, 'executeCommand', return_value=[None])
-    results, errors = create_indicators_loop(indicators=indicators)
+    results, errors = create_indicators_loop(args, indicators)
+    assert errors == []
+    assert results.readable_output == 'Create Indicators From STIX: 2 indicators were created.'
+
+
+def test_create_indicators_loop_w_args(mocker):
+    """
+    Given:
+        - A collection of indicators in XSOAR Format
+
+    When:
+        - Parsing stix indicators.
+
+    Then:
+        - Validate the indicators extract without errors.
+    """
+    args = {
+        'add_context': 'true',
+        'tags': 'tag1,tag2'
+    }
+
+    with open('test_data/expected_result_example3.json') as json_f:
+        indicators = json.load(json_f)
+    mocker.patch.object(demisto, 'executeCommand', return_value=[None])
+    results, errors = create_indicators_loop(args, indicators)
     assert errors == []
```

Comment:
please add a test for `create_indicators_loop` with tags and `add_context=true` and check the context created and the tags added

Reply:
Added as part of commit.